packages/file-search/src/browser/quick-file-open.ts

```diff
@@ -58,6 +60,8 @@ export class QuickFileOpenService implements QuickOpenModel, QuickOpenHandler {
     protected readonly messageService: MessageService;
     @inject(FileSystemPreferences)
     protected readonly fsPreferences: FileSystemPreferences;
+    @inject(FileQuery)
+    protected readonly currentFileQuery: FileQuery;
```

Comment:
```suggestion
    protected readonly fileSearchQuery: FileSearchQuery;
```

Reply:
Done, reused this name with the new implementation.